app/components/global-navigation/search.js

```diff
@@ -56,6 +56,17 @@ export default Component.extend({
 
 		// key: query string, value: Array<SearchSuggestionItem>
 		this.cachedResults = {};
+		this.setScopeMessage(this.getScope());
+	},
+
+	setScopeMessage(scope) {
+		if (scope === 'cross-wiki') {
+			this.set('model.placeholder-active.key', 'global-navigation-search-placeholder-all-wikis');
```

Comment:
small thing, please try to keep consistent names, here you have `cross-wiki` vs `all-wikis`

Reply:
also you could introduce some enumeration for those hardcoded keys